sys/net/routing/aodvv2/reader.c

```diff
@@ -616,6 +618,7 @@ static enum rfc5444_result _cb_rerr_blocktlv_addresstlvs_okay(struct rfc5444_rea
 
     VDEBUG("%s()\n", __func__);
     VDEBUG("\tmessage type: %d\n", cont->type);
+    /* cppcheck-suppress Unused variable as VDEBUG is needed for better debugging output. */
```

Comment:
oops!  forgot to undo the changes while editing , sending the updated changes in a minute.